test/testcafe/spec/ReEnrollAuthenticatorWarningPasswordView_spec.js

```diff
@@ -65,21 +66,21 @@ async function setup(t) {
   [ 'Your password will expire later today', mockExpireToday ],
   [ 'Your password is expiring soon', mockExpireSoon ],
 ].forEach(([ formTitle, mock ]) => {
-  test
+  test.meta('v3', false)
     .requestHooks(logger, mock)('Should have the correct labels - expire in days', async t => {
       const passwordExpiryWarningPage = await setup(t);
       await t.expect(passwordExpiryWarningPage.getFormTitle()).eql(formTitle);
-      await t.expect(passwordExpiryWarningPage.getSaveButtonLabel()).eql('Change Password');
+      await t.expect(passwordExpiryWarningPage.changePasswordButtonExists()).eql(true);
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('Password requirements:');
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('At least 8 characters');
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('An uppercase letter');
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('A lowercase letter');
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('A number');
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('No parts of your username');
       await t.expect(passwordExpiryWarningPage.getRequirements()).contains('Your password cannot be any of your last 4 passwords');
```

Comment:
I had to disable this test for v3 because we removed server side password requirements from the view. 

Should I instead add a conditional statement here to allow the other assertions to pass (I tested the locally and they do pass) @shawyu-okta

Reply:
Hmm good question - I think normally we don't want to change the test assertions dependent on which version of the widget due to our goal of parity but I think in this case we did make the call that this was not a parity feature we wanted, so I think that's OK if we mark it as such with a short explanation.